### ISSUE TITLE
Empty table header cells in data tables in CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10133]
## Description
There are data tables within the CMS that are missing content within the table header cells.  Empty table header cells create confusion for users as they may not be able to fully understand the content in the associated table data cells.

## Additional Context
List of tables where this was found (may not be the full list of all tables within the CMS):

- VAMC Facility Health Service Edit screen: Other Phone Numbers Table
- VAMC Facility Non-Clinical Service Edit screen: Other Phone Numbers Table
- VAMC Facility Non-Clinical Service Edit screen: Hours Table
- VAMC System Billing and Insurance Edit screen: Hours Table
- VAMC System Operating Status Edit screen: System-wide alerts Table
- VAMC System Operating System Edit screen: Facility Operating statuses Table

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/183929658-c3151095-46ec-48ab-a65c-d44ce8309f4c.png)

## Accessibility Standard

WCAG version 2.0 A, [Criterion 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

## Acceptance Criteria
- [ ] Technical review for adding scope to tables (one update for all or individual updates)
- [ ] Change management consulted
- [ ] Implementation ticket(s) created

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by laflannery** [comment]
Updated description to indicate which ones where remediated by [update to office_hours module ](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13956)

**Comment by laflannery** [comment]
@ndouglas Does this need to be broken up into product tickets? It's not just one field type or markup that generates tables this way and would be one ticket?

**Comment by ndouglas** [comment]
@laflannery 

For Benefit Hub(s), are you referring to this interface when you select Benefit Hubs?

![Screenshot 2023-10-31 at 11 30 05 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/1318579/069471d9-12db-4a10-b677-c88d0a2a9d96)

Or are you referring to the field as it appears "at rest"?

![Screenshot 2023-10-31 at 11 30 46 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/1318579/e77ec6df-1ec9-45a5-bdea-2036ceacaae6)

**Comment by laflannery** [comment]
At rest, the modal doesn't actually have headers (which is neither here nor there at the moment but possibly something that should have another ticket)

**Comment by ndouglas** [comment]
@laflannery It seems most of these can probably be fixed in a hook that injects "Weight" (or something else?) in the table header for certain fields. I'll defer to @edmund-dunn and @tonytaylor on implementation, though. I think this can all remain in one ticket, unless either of them disagree.

**Comment by laflannery** [comment]
@ndouglas thank you for looking! It's definitely a lower priority but I started going through a lot of tickets today so I just wanted to split this up if I needed to. I really appreciate you taking the time!

**Comment by srancour** [comment]
This is something that still exists and should still be worked on.

**Comment by gracekretschmer-metrostar** [comment]
@JakeBapple I would like you to do a technical review of this work and @srancour could you do the UX/IA to determine what content should say? 

Note, this is work that can be done when you have extra time and have finished any work related to sprint goals.

**Comment by bapplejax** [comment]
We may be able to add a site-wide markup solution to empty tds and ths by conditionally adding the aria-label="No value" to any cell in a table with no value. That work would be small in itself but the testing would need to be larger in scope and take up the bulk of the time. Estimated actual dev work would be about an hour to make the changes and spot check locally.
